sopel/bot.py

```diff
@@ -174,10 +183,16 @@ def write(self, args, text=None):  # Shim this in here for autodocs
         Newlines and carriage returns ('\\n' and '\\r') are removed before
         sending. Additionally, if the message (after joining) is longer than
         than 510 characters, any remaining characters will not be sent.
+
+        :param collections.Iterable args: an iterable of strings, which will be joined by spaces
```

Comment:
You'd need to use `:type:` lines also, but it's possible to link things like this to the Python glossary:

```
:param args: command arguments, which will be joined by spaces
:type args: :term:`iterable`
```

That produces the type hint "([iterable](https://docs.python.org/3/glossary.html#term-iterable))" for `args`, linking to the term in Python's own docs. Feel free to use this trick all over the place! There are many references to such things in this PR.

Reply:
Thanks for this hint - I'm reasonably new to Sphinx so this is a learning experience. I've used `:term:` in a couple of places in the latest revision!